readme.md

```diff
@@ -1,7 +1,7 @@
 # Frontend practice with catalog page
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://JOWISSA.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://JOWISSA.github.io/layout_catalog/report/html_report/)
```

Comment:
please add this links in your PR description

Links doesn't work, did you ran `rpm run deploy`